.travis.yml

```diff
@@ -88,8 +88,8 @@ deploy:
     env: wire-webapp-rc
     bucket_name: wire-webapp-builds
     on:
-      repo: wireapp/wire-webapp
-      branch: staging
+      tags: true
+      condition: $TRAVIS_TAG =~ *-staging*
```

Comment:
But does this really work with Travis?

Reply:
Documentation sounds very promising: https://docs.travis-ci.com/user/deployment#conditional-releases-with-on